packages/atomic/src/components/atomic-pager/atomic-pager.tsx

```diff
@@ -29,52 +31,78 @@ export class AtomicPager implements InitializableComponent {
   @BindStateToController('pager')
   @State()
   private pagerState!: PagerState;
-  @State() public error!: Error;
+  @BindStateToI18n()
+  @State()
+  private strings = {
+    pagination: () => this.bindings.i18n.t('pagination'),
+    previous: () => this.bindings.i18n.t('previous'),
+    next: () => this.bindings.i18n.t('next'),
+    pageNumber: (page: number) => this.bindings.i18n.t('pageNumber', {page}),
+  };
+  @State() error!: Error;
+
+  /**
+   * Specifies how many page buttons to display in the pager.
+   */
+  @Prop() numberOfPages = 5;
+  /**
+   * Specifies whether the **Previous** and **Next** buttons should appear at each end of the pager when appropriate.
+   */
+  @Prop() enableNavigationButtons = true;
 
   public initialize() {
-    this.pager = buildPager(this.bindings.engine);
+    this.pager = buildPager(this.bindings.engine, {
+      options: {numberOfPages: this.numberOfPages},
+    });
   }
 
-  private get backButton() {
-    if (!this.pagerState.hasPreviousPage) {
-      return null;
-    }
-
-    const icon = '<';
+  private buildButton(options: {
+    part: string;
+    disabled: boolean;
+    ariaLabel: string;
+    callback: () => void;
+    icon: string;
+  }) {
     return (
       <li>
         <button
-          part="back-button"
-          aria-label="Previous page"
-          onClick={() => {
-            this.pager.previousPage();
-          }}
+          part={options.part}
+          class={`text-secondary ${
+            options.disabled ? 'opacity-50 cursor-not-allowed' : ''
+          }`}
+          disabled={options.disabled}
+          aria-hidden={options.disabled}
```

Comment:
I'm not sure if I'd use `aria-hidden` here, given that the button is disabled anyways. I haven't tested, but I'm guessing setting `aria-hidden` to `true` on an element focused by accessibility tools may cause issues.